pkg/agent/multicast/mcast_controller.go

```diff
@@ -263,10 +263,12 @@ type Controller struct {
 	// the group ID in OVS for group which IGMP queries are sent to
 	queryGroupId binding.GroupIDType
 	// nodeGroupID is the OpenFlow group ID in OVS which is used to send IGMP report messages to other Nodes.
-	nodeGroupID binding.GroupIDType
+	nodeGroupID           binding.GroupIDType
+	forwardInboundGroupID binding.GroupIDType
```

Comment:
Add comment for `forwardInboundGroupID`.

Reply:
updated